### PR TITLE
CVE-2022-25620

### PR DESCRIPTION
**Pull request by ulkuderner**
CVE-2022-25620.json

### CODE REVIEW

**Comment by cve-team** [comment]
ulkuderner is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2022/25xxx/CVE-2022-25620.json [hunk 1]
```diff
@@ -1,18 +1,105 @@
 {
```
Comment (cve-team) [inline 2022/25xxx/CVE-2022-25620.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/25xxx/CVE-2022-25620.json:1]: ('Invalid control character at: line 47 column 73 (char 1518)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @ulkuderner,

Once the errors in the file have been addressed we can proceed with publishing the information to the main list.

Thanks,
CVE Content Team

File: 2022/25xxx/CVE-2022-25620.json (hunk 1 above)
Comment (cve-team) [inline 2022/25xxx/CVE-2022-25620.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/25xxx/CVE-2022-25620.json:1]: ('Invalid control character at: line 47 column 73 (char 1518)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/25xxx/CVE-2022-25620.json [hunk 2]
```diff
@@ -1,18 +1,100 @@
...
```
Comment (cve-team) [inline 2022/25xxx/CVE-2022-25620.json:1]: MISC or CONFIRM, but name != url for reference.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Pull request merged**